packages/cli/src/ui/components/Footer.tsx

```diff
@@ -39,10 +41,15 @@ export const Footer: React.FC<FooterProps> = ({
   showMemoryUsage,
   promptTokenCount,
   nightly,
+  display,
 }) => {
   const limit = tokenLimit(model);
   const percentage = promptTokenCount / limit;
 
+  if (display.footer === false) {
```

Comment:
one concern is that users of this setting will not be able to tell when errors have occurred. I would be more comfortable with the option if we still showed the number of errors that had occurred even when displayFooter is false.